### PR TITLE
Update README.md

### LINKED ISSUE #55: Add info about GPU and CUDA requirements in readme

### PR DESCRIPTION
**Pull request by jammy270**
Added a line about setting up a GPU

Fix #55

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -22,6 +22,7 @@ It may also work on other platforms.
 -   If you prefer conda, you may install it from [here](https://www.anaconda.com/products/individual)
 
 ## Installation Instructions
+To setup your GPU for use with tensorflow, please follow the tensorflow guide for [GPU support](https://www.tensorflow.org/install/gpu). Alternatively, you can also use a tensorflow [Docker container](https://www.tensorflow.org/install/docker).
```
Comment (ashishsingh18) [inline README.md:25]: @jammy270 Do we need the last sentence about the docker? How would that help with local install?

Comment (jammy270) [reply README.md]: I removed that line.

**Review by jammy270 — COMMENTED** [review]
Removed the line about docker container

**Review by ashishsingh18 — APPROVED** [review]
looks good.

**Pull request merged**